packages/core/src/resolver.js

```diff
@@ -1,10 +1,13 @@
 import Module from 'module'
 import { resolve, join } from 'path'
 import fs from 'fs-extra'
+import consola from 'consola'
 import esm from 'esm'
 
 import { startsWithRootAlias, startsWithSrcAlias } from '@nuxt/utils'
 
+const logger = consola.withTag('@nuxt/resolver')
```

Comment:
We don't use scopped loggers anywhere else. If we are going to use them i suggest continue it in another PR and adding a unified helper in core or utils for access (second part is just an idea)

Reply:
@pi0 maybe something like https://github.com/nuxt/fabula/blob/master/src/logging.js?

Reply:
Agee. Transforming it back to normal `consola` but add it to the feature/idea backlog

Reply:
@galvez Yes. Exactly. Consola 3 uses lightweight tagged loggers using Proxy.

Reply:
@manniL Thanks :)